.ci/pack

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+NAME=redash
+VERSION=$(jq -r .version package.json)
+FULL_VERSION=$VERSION+b$CIRCLE_BUILD_NUM
```

Comment:
We should probably look into that `$CIRCLE_BUILD_NUM` variable at some future point, since we're not using CircleCI any more.

Not super relevant for this PR though. :wink: